provision/filter_plugins/aws_inv.py

```diff
@@ -8,7 +8,9 @@
     from configparser import ConfigParser
 except ImportError:
     from ConfigParser import ConfigParser
-sys.path.append(os.getcwd())
+filepath = os.path.realpath(__file__)
```

Comment:
@herlo : I feel adding it to easy_install path is great idea. In that case we need to have a common path/folder for all the packages we write from here on. Something like linchpin_pkgs/packages. We can also have a parameter on linchpin_config.yml  
However , packaging both ansible modules and filter classes together seems not fair , because ansible packages have different purpose when compared to filter classes. I would suggest to maintain them separately.